crates/nostr-sdk/src/client/mod.rs

```diff
@@ -214,16 +221,17 @@ impl Client {
 
     /// Get NIP46 uri
     #[cfg(feature = "nip46")]
-    pub fn nostr_connect_uri(
+    pub async fn nostr_connect_uri(
```

Comment:
Isn't turning a synchronous function into an asynchronous one in the public API a breaking change?